docker-compose.yml

```diff
@@ -11,18 +11,6 @@ services:
     ports:
       - "5432:5432"
 
-  pgweb:
```

Comment:
This is in the compose stack intentionally, but I'm not opposed to removing it in a separate PR if the community is open to it.